### PR TITLE
Tweak feedbacks

### PR DESCRIPTION
**Pull request by shhdgit**

### CODE REVIEW

File: src/components/MDXComponents/LearningPath.tsx [hunk 1]
```diff
@@ -250,13 +250,23 @@ export function LearningPath(props: {
...
-              width: "3rem",
-              height: "3rem",
+              width: {
+                xs: "1.6rem",
+                md: "3rem",
+              },
+              height: {
+                xs: "1.6rem",
+                md: "3rem",
+              },
```
Comment (Yuiham) [inline src/components/MDXComponents/LearningPath.tsx:260]:
```suggestion
              width: "3rem",
              height: "3rem",
              display: {
                 xs: "none",
                 md: "initial"
              }
```

File: src/styles/docTemplate.css [hunk 2]
```diff
@@ -27,6 +27,10 @@
       list-style: initial;
     }
 
+    li {
+      margin-top: 8px;
+    }
```
Comment (Yuiham) [inline src/styles/docTemplate.css:32]: Why?

File: src/components/MDXComponents/DocHome.tsx [hunk 3]
```diff
@@ -110,25 +111,21 @@ export function DocHomeContainer(
           {!archive && ctaLink && (
             <Button
               variant="contained"
-              color="primary"
+              href={ctaLink}
+              target="_blank"
+              referrerPolicy="no-referrer-when-downgrade"
```
Comment (Yuiham) [inline src/components/MDXComponents/DocHome.tsx:116]:
```suggestion
              className="cta-link"
              href={ctaLink}
              target={ctaLink.startWith("http") ? "_blank" : undefined}
```

File: src/components/MDXComponents/DocHome.tsx [hunk 4]
```diff
@@ -110,25 +111,21 @@ export function DocHomeContainer(
...
-                "& > a.button": {
-                  color: "white",
-                  "&:hover": {
-                    textDecoration: "none",
-                  },
+                background: "var(--primary-color) !important",
+                textDecoration: "none !important",
+                color: "white !important",
+                "&:hover": {
+                  backgroundColor: "var(--primary-color-dark) !important",
```
Comment (Yuiham) [inline src/components/MDXComponents/DocHome.tsx:124]: Move the css to its parent without `!important` using `& > .cta-link`.

File: src/components/Layout/HeaderAction.tsx [hunk 5]
```diff
@@ -288,16 +289,18 @@ const TiDBCloudBtnGroup = () => {
           "aria-labelledby": "idb-cloud-menu-button",
         }}
       >
-        <MenuItem onClick={handleClose}>
+        <MenuItem
+          onClick={handleClose}
```
Comment (Yuiham) [inline src/components/Layout/HeaderAction.tsx:293]: Move gtm tracking into `onClick`.

**Review by Yuiham — CHANGES_REQUESTED** [review]

File: src/components/MDXComponents/DocHome.tsx [hunk 6]
```diff
@@ -108,28 +109,30 @@ export function DocHomeContainer(
...
           </Typography>
           {!archive && ctaLink && (
-            <Button
-              variant="contained"
-              color="primary"
-              size="large"
+            <Box
               sx={{
-                "& > a.button": {
+                "& a.button": {
```
Comment (Yuiham) [inline src/components/MDXComponents/DocHome.tsx:114]: Raise priority

```suggestion
                "& > a.button": {
```

File: src/components/MDXComponents/DocHome.tsx [hunk 7]
```diff
@@ -108,28 +109,30 @@ export function DocHomeContainer(
...
               }}
             >
-              <a
-                className="button"
+              <Button
+                variant="contained"
                 href={ctaLink}
-                target="_blank"
+                target={ctaLink.startsWith("http") ? "_blank" : undefined}
                 referrerPolicy="no-referrer-when-downgrade"
```
Comment (Yuiham) [inline src/components/MDXComponents/DocHome.tsx:128]: We can remove this line for CTA tracking.

File: src/styles/docTemplate.css (hunk 2 above)
Comment (shhdgit) [reply src/styles/docTemplate.css:32]: be close to Figma.

from:
image
->
image

Figma:
image

Comment (Yuiham) [inline src/styles/docTemplate.css:32]:
```suggestion
    li:not(:first-of-type) {
      margin-top: 8px;
    }
```

File: src/components/MDXComponents/DocHome.tsx [hunk 8]
```diff
@@ -241,8 +243,6 @@ export function DocHomeContainer(
                           lineHeight: "1.25rem",
                           borderLeft: `1px solid transparent`,
                           paddingLeft: `0.5rem`,
-                          paddingTop: "0.25rem",
-                          paddingBottom: "0.25rem",
```
Comment (Yuiham) [inline src/components/MDXComponents/DocHome.tsx:245]: Can we keep the padding?

**Review by Yuiham — APPROVED** [review]

**Pull request merged**